### DISCUSSION TITLE
How to use VisualizerWithEditing() with WebRTC

### DISCUSSION THREAD

**Discussion by kekomatsu** [isl-org/Open3D#5871] [Q&A] [answered]
Hello, 
I want to use point picker of o3d.visualization.VisualizerWithEditing() and want to run that in WebRTC.
I tried to set enable_webrtc() and VisualizerWithEditing() but not work.
`o3d.visualization.webrtc_server.enable_webrtc()`
`vis = o3d.visualization.VisualizerWithEditing()`
`vis.create_window()`
`vis.add_geometry(pcd)`
`vis.run() `

I want to use "Select correspondences" on this page.
http://www.open3d.org/docs/release/tutorial/visualization/interactive_visualization.html

Is there any way to use VisualizerWithEditing() with WebRTC mode?

**Comment by ssheorey** [comment, marked as answer]
This is not possible. 

Instead, you can select points with O3DVIsualizer / draw() functions.
